final_task/rss_reader/parse_news.py

```diff
@@ -0,0 +1,28 @@
+from bs4 import BeautifulSoup
+import feedparser
+import json
+
+# check if there is media content
```

Comment:
Docstrings should be like that  `"""Text."""` at the first line of function

Reply:
Fixed